packages/hagrid/hagrid/cli.py

```diff
@@ -3153,7 +3153,7 @@ def create_land_cmd(verb: GrammarVerb, kwargs: TypeDict[str, Any]) -> str:
             # subprocess.call("docker rm `docker ps -aq` --force", shell=True) # nosec
 
             if "prune_vol" in kwargs:
-                return "docker rm `docker ps -aq` --force && docker volume prune"
+                return "docker rm `docker ps -aq` --force && docker volume prune -f"
```

Comment:
I don't think this is a good idea. If you ran this on your desktop and it removed all your docker volumes including for other applications you are running without asking you, you would be 🤬

Reply:
I need this fix for an issue in dev, so im going to merge but lets discuss this in the QA process.